### DISCUSSION TITLE
Please add CARDANO ADA Full Wallet support

### DISCUSSION THREAD

**Discussion by gushani** [LedgerHQ/ledger-live-desktop#4010] [Coin/Token request]
#### Ledger Live Version and Operating System

- tested on Ledger Live **version_here**
- Platform and version: **e.g. Mac 10.13.5 / Windows 10 / ..**

**Comment by twinko** [comment]
Hello Ledger Support,

how comes ADA is not supported by Ledger Live? It's 5th place in overall M-Cap? Would love to see Support for ADA.
Moreover it would would give you a lead from competitors ;)

Thank you for your work and great product.
![grafik](https://user-images.githubusercontent.com/11973158/120630794-70ea2200-c467-11eb-8036-665af06e7277.png)

**Comment by KazW** [comment]
Hi, I'm looking to build account support for ADA and Cardano tokens into the Ledger Live desktop and mobile apps. I've submitted a Cardano Catalyst Proposal to secure funding to build this feature out and it's been very well received by the community. See the proposal [here](https://cardano.ideascale.com/a/dtd/Ledger-Live-Support/367170-48088).

I'd like to pursue building this, but before finalizing the proposal, I'd like to know if my contributions would be accepted into the Ledger Live apps.

Would the code for this be accepted into the apps if I followed all the style and testing conventions in the apps, and built it to the same standard as other cryptos with account support?

Ping: @ledgerlive @cthepot-ledger

**Comment by johnpc** [comment]
There are several other threads requesting ADA support:

* https://github.com/LedgerHQ/ledger-live-desktop/discussions/3742
* https://github.com/LedgerHQ/ledger-live-desktop/discussions/3420
* https://github.com/LedgerHQ/ledger-live-desktop/discussions/3653

If you add up the votes across these issues it's easily the most upvoted feature request here. Still no response from @ledgerlive representatives

**Comment by johnpc** [comment]
It's happening: https://twitter.com/Ledger/status/1469240129319579648
